map/src/drawer/components/poi/PoiTypesDialog.jsx

```diff
@@ -100,6 +100,23 @@ export default function PoiTypesDialog({ dialogOpen, setDialogOpen }) {
         return categories.filter((category) => category !== 'user_defined_other' && category !== 'osmwiki');
     }
 
+    function removeUnusedFilters(filters) {
+        filters = filters.filter((f) => f !== 'routes');
```

Comment:
Do you really need to modify function parameter and then return it also? If not, just use **return filters.filter(...)**